src/sentry/sentry_apps/api/serializers/request_v2.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+from collections.abc import Mapping, MutableMapping, Sequence
+from typing import Any
+
+from sentry.api.serializers import Serializer
+from sentry.hybridcloud.services.organization_mapping import organization_mapping_service
+from sentry.sentry_apps.models.sentry_app import SentryApp
+
+
+class RequestSerializer(Serializer):
```

Comment:
This name is a bit generic, could we make it more specific to SentryApp Webhook Requests?